lib/ask_export.rb

```diff
@@ -2,11 +2,8 @@
 require "active_support/time"
 Time.zone = "Europe/London"
 
-require_relative "ask_export/csv_builder"
-require_relative "ask_export/csv_splitter"
-require_relative "ask_export/partner_notifier"
-require_relative "ask_export/runner"
-require_relative "ask_export/survey_response_fetcher"
+files = Dir.glob(File.join(__dir__, "ask_export/*.rb")).sort
```

Comment:
Do we sort them so that any module definitions come first?

Reply:
Nope - it was too appease the linter. I've moved this to a single line since it stopped flagging on it.

But the complaint is https://docs.rubocop.org/en/latest/cops_lint/#lintnondeterministicrequireorder and I imagine this rings bells for you :-)